### PR TITLE
Refactor es6

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Refactored vars to use es6 and also changed string concatenations to template literals.

Let me know if these changes are okay, I also ran `gulp format` and seemed to get a lot of strange changes, let me know if I should remove that.

### CODE REVIEW

**Comment by yuchi** [comment]
I ❤️ this `const`-by-default thing.

**Comment by eduardolundgren** [comment]
LGTM, did you run `gulp format lint`? Thanks!

**Comment by bryceosterhaus** [comment]
@eduardolundgren yeah the second commit is the result from running `gulp format lint` but it seemed to produce a strange output.

File: packages/metal-component/src/events/events.js [hunk 1]
```diff
@@ -58,9 +58,9 @@ export function getComponentFn(component, fnName) {
 	if (isFunction(component[fnName])) {
 		return component[fnName].bind(component);
 	} else {
-		console.error('No function named "' + fnName + '" was found in the ' +
-			'component "' + getFunctionName(component.constructor) + '". Make ' +
-			'sure that you specify valid function names when adding inline listeners.'
+		console.error(`No function named ${fnName} was founc in the component
```
Comment (mairatma) [inline packages/metal-component/src/events/events.js:61]: There were a couple of minor errors here when converting to the new format:
* `found` became `founc`
* `valid` became `validf`

These are the only ones I found though, so I can just fix it myself and merge everything now :)

**Comment by mairatma** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by mairatma — COMMENTED** [review]
Yes, I've stopped using `gulp format` for a while because of the strange output, waiting for when I would have time to investigate why this was happening and fix it. It's been a long time already, and in the end we're now missing some real fixes from `gulp format` by not using it, so I think it's best to just go back to using it, even if it's weird. Once it's fixed we'll do another format change. So let's keep these changes for now.

**Comment by mairatma** [comment]
This has been [merged](https://github.com/metal/metal.js/commits/master), thanks!

**Pull request closed without merging**